openscap_report/report_generators/html_templates/evaluation_characteristics.html

```diff
@@ -1,9 +1,81 @@
 {# Copyright 2022, Red Hat, Inc. #}
 {# SPDX-License-Identifier: LGPL-2.1-or-later #}
-<div class="pf-c-accordion pf-m-display-lg">
+<div>
+    <ul class="pf-c-data-list" role="list">
+
+        <li class="pf-c-data-list__item">
+            <div class="pf-c-data-list__item-row">
+                <div class="pf-c-data-list__item-content">
+                    <div class="pf-c-data-list__cell"><b>Evaluation target:</b></div>
+                    <div class="pf-c-data-list__cell">
+                        {{- report.scan_result.target -}}
+                    </div>
+                </div>
+            </div>
+        </li>
+
+        <li class="pf-c-data-list__item">
+            <div class="pf-c-data-list__item-row">
+                <div class="pf-c-data-list__item-content">
+                    <div class="pf-c-data-list__cell"><b>Profile ID:</b></div>
+                    <div class="pf-c-data-list__cell">
+                        {{- report.profile_info.profile_id -}}
+                    </div>
+                </div>
+            </div>
+        </li>
```

Comment:
This `<li>` element repeats multiple times here. Can it be extracted to a macro?